amlip_cpp/src/cpp/node/ParentNode.cpp

```diff
@@ -31,7 +34,24 @@ ParentNode::ParentNode(
         types::NodeKind node_kind,
         types::StateKind initial_state,
         uint32_t domain_id)
-    : participant_(std::make_unique<dds::Participant>(name, dds::Participant::default_participant_qos(), domain_id))
+    : participant_(std::make_unique<dds::Participant>(name, dds::utils::default_domain_participant_qos(name), domain_id))
```

Comment:
This constructor must call the other with the defaulted argument.
This way, the constructor logic is stored in only one function
`    : ParentNode(name, node_kind, initial_state, domain_id, dds::utils::default_domain_participant_qos(name))`